src/base/ddl_log.c

```diff
@@ -429,6 +448,56 @@ logddl_get_jsp_mode ()
   return jsp_mode;
 }
 
+bool
+logddl_get_sql_text (const char *file, int start_line, int end_line, char *sql_text)
```

Comment:
You defined **logddl_get_sql_text ()** here, and I can't find where to call this function.

Reply:
Thank you, This is an unused function, so I will delete it.